Library/NumberFormatter.swift

```diff
@@ -0,0 +1,67 @@
+// swiftlint:disable line_length
+import Foundation
+
+final class AttributedNumberFormatter: NumberFormatter {
+  // MARK: - Properties
+
+  var defaultAttributes: String.Attributes = [:]
+  var currencySymbolAttributes: String.Attributes = [:]
+  var decimalSeparatorAttributes: String.Attributes = [:]
+  var fractionDigitsAttributes: String.Attributes = [:]
+
+  // MARK: - Attributed string
+
+  override func attributedString(for obj: Any, withDefaultAttributes attrs: [NSAttributedString.Key: Any]? = nil) -> NSAttributedString? {
```

Comment:
Documentation indicates that this has to be overridden.

Doing so makes it pretty trivial for us to set various attributes to various portions of the output string (including currency symbol, fraction digits and decimal separator)